docs/development/release.md

```diff
@@ -84,17 +89,21 @@ If you created a new release plan in step one, you will need to create a new bam
 
 * In the Cumulus Core project (<https://ci.earthdata.nasa.gov/browse/CUM-CBA>), click Actions -> Configure Plan
 
-* Scroll to the bottom of the branch list and click `Create Plan Branch`
+* Scroll to the bottom of the branch list and click `Create Plan Branch`.
+
+* Click `Create plan branch manually`.
+
+* Add the values in that list. Choose a display name that makes it *very* clear this is a deployment branch plan. `Release (branch name)` seems to work well. *Make sure* you select the correct branch
 
-* Click `Create plan branch manually`
+* **Important** Deselect Enable Branch - if you do not do this, it will immediately fire off a build.
 
-* Add the values in that list.   Choose a display name that makes it *very* clear this is a deployment branch plan.    `Release (branch name)` seems to work well.    *Make sure* you select the correct branch
+* **Immediately** go to plan configuration on the `Branch Details` tab, and enable `Change trigger`.  Set the `Trigger type` to manual, this will prevent commits to the branch from triggering the build plan.
 
-* **Important** Deselect Enable Branch - if you do not do this, it will immediately fire off a build
+* Go to the branch plan and set GIT_PR, USE_NPM_PACKAGES, SKIP_AUDIT and PUBLISH_FLAG to true.  Select a DEPLOYMENT appropriate for the release (defaults to last committer). This should be `cumulus-from-npm-tf` *except* in special cases such as incompatible backport branches.
 
-* **Immediately** go to plan configuration on the `Branch Details` tab, and enable `Change trigger`.  Set the `Trigger type` to manual, this will prevent commits to the branch from triggering the build plan
+* Enable the branch from the `Branch Details` page.
 
-* Go to the branch plan and set GIT_PR, USE_NPM_PACKAGES, SKIP_AUDIT and PUBLISH_FLAG to true.  Select a DEPLOYMENT appropriate for the release (defaults to last committer). This should be `cumulus-from-npm` *except* in special cases such as incompatible backport branches.
+* Run the branch using the `Run` button in the top right.
```

Comment:
These last two steps were kind of missing as if Bamboo will somehow start itself when previous steps are clearly spelled out.